mainframer

```diff
@@ -134,6 +157,25 @@ function executeRemoteCommand {
 	echo ""
 }
 
+function sendServerCommand {
```

Comment:
??

Reply:
This should not be here... I was testing some alternative to pass on SIGINT without interactive ssh, gonna remove it.